src/sentry/api/serializers/models/sentry_app_component.py

```diff
@@ -18,14 +18,12 @@ def serialize(self, obj, attrs, user):
 
 
 class SentryAppAlertRuleActionSerializer(Serializer):
-    def serialize(self, obj, attrs, user, install, **kwargs):
+    def serialize(self, obj, attrs, user, **kwargs):
+        install = kwargs.get("install")
```

Comment:
You should have a default value in case someone tries using this serializer without passing `install`.

Reply:
Will address in https://github.com/getsentry/sentry/pull/28659/